extensions/jsdbg/toolbar/toolbar.js

```diff
@@ -221,6 +221,15 @@ Loader.OnLoad(function () {
 
         toolbar.appendChild(threadSelectorPane);
 
+        var continueExecution = document.createElement("button");
```

Comment:
We should add a "step" button as well. I think it will be similarly valuable.

Reply:
Yeah that sounds good, but as a seperate change perhaps.